client/blocks/reader-full-post/style.scss

```diff
@@ -257,6 +257,7 @@
 		}
 
 		.author-compact-profile__site-link {
+			flex: 1;
```

Comment:
I don't think we should use the shorthand anymore because of how we recently ran into IE11 issues.  
As done in https://github.com/Automattic/wp-calypso/pull/10025 can you change this to `flex: 1 0 auto;`